hazelcast/src/main/java/com/hazelcast/config/tpc/TPCEngineConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.hazelcast.config.tpc;
```

Comment:
I find this class a bit confusing. 

There is a TpcEngine.Configuration, and now also a TPCEngineConfig.

We also need to think about the future where there is only 'alto'. Perhaps we should just call this class AltoConfiguration? What do you think?

Reply:
I think TpcEngine.Configuration as internal and TPCEngineConfig as public. So the previous includes everything that can be configured by a dev and latter includes only what we allow for users.

Changing naming makes sense. I used `TPCEngineConfig` name because that is where these configurations live (tpc engine). I can change this but then what about `TPCSocketConfig`?